app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherActivity.kt

```diff
@@ -208,6 +209,13 @@ class TabSwitcherActivity : DuckDuckGoActivity(), TabSwitcherListener, Coroutine
             }
         }
 
+        lifecycleScope.launch {
```

Comment:
what’s this for?

Reply:
This prevented deleting of the last tab but after Ana’s comment I got rid of it.